controllers/user.js

```diff
@@ -31,17 +32,43 @@ async function add(req, res) {
   });
 }
 
-
 async function verify(req, res) {
   const verifiedEmailHash = await DB.verifyEmailHash(req.query.token);
 
+  const unsafeBreachesForEmail = await HIBP.getUnsafeBreachesForEmail(
+    sha1(verifiedEmailHash.email),
+    req.app.locals.breaches
+  );
+
+  if(unsafeBreachesForEmail) {
+    unsafeBreachesForEmail.forEach((breach) => {
+      breach.BreachDate = new Date(breach.BreachDate).toLocaleString("en-US", {year: "numeric", month: "long", day: "numeric"});
+      breach.DataClasses = breach.DataClasses.join(", ");
+    });
+  }
+
+  const unsubscribeUrl = EmailUtils.unsubscribeUrl(verifiedEmailHash);
+  const serverUrl = req.app.locals.SERVER_URL;
+  await EmailUtils.sendEmail(
+    verifiedEmailHash.email,
+    "Your Firefox Monitor report",
+    "report",
+    {
+      email: verifiedEmailHash.email,
+      date: new Date().toLocaleString("en-US", {year: "numeric", month: "long", day: "numeric"}),
+      unsafeBreachesForEmail,
+      TIPS,
+      unsubscribeUrl,
+      serverUrl,
+    }
+  );
+
   res.render("confirm", {
     title: "Firefox Monitor: Subscribed",
     email: verifiedEmailHash.email,
   });
 }
 
-
```

Comment:
Sorry to be so nit-picky but let's not ⚡️ these lines. 2 lines between each request controller helps when scan-scrolling the file.